andela/mervels-bn-backend
# #168131424 Integrate requests table to the backend

#### What does this PR do?
Integrate requests table to the backend
#### Description of Task to be completed?
- Add the requests table
- Write the endpoint that allows a user to retrieve his/her requests
#### How should this be manually tested?
- Clone this repo
- Checkout to branch `ft-implement-requests-table-168131424`
- Install dependencies i.e. `npm install`
- Run `npm test`
- Run the application with `npm run start:dev` and after you successfully log in make a `GET` request to `api/v1/requests/my-requests` to retrieve your requests
- As an alternative, you can the Heroku review app to test this: `http://mervels-bn-backend-stagi-pr-24.herokuapp.com/api/v1/requests/my-requests`
- After you do the request you should see a response that looks like the one in screenshots
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131424](https://www.pivotaltracker.com/story/show/168131424)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/50402526/65060847-f0c11780-d978-11e9-8fc6-6b4de444e80e.png)

#### Questions:
N/A
